modules/apps/expando/expando-test/src/com/liferay/expando/test/ExpandoConverterUtilTest.java

```diff
@@ -12,10 +12,10 @@
  * details.
  */
 
-package com.liferay.portlet.expando.util;
+package com.liferay.expando.test;
```

Comment:
@slnn The directory structure is wrong. A unit test should be in `src/test/java/....`. You missed the `test/java` part.

Reply:
@slnn Also, unit tests are usually put in the `src/test/java` in the same module of the tested class. So you can put this test in `expando-api`.

Reply:
Done